.github/workflows/metrics-comparison.yml

```
@@ -41,7 +41,7 @@ jobs:
 
       - name: Compare performance with trunk
         if: github.event_name == 'pull_request'
-        run: cd scripts/compare-perf && npm ci && npm run compare -- perf $GITHUB_SHA trunk --tests-branch $GITHUB_SHA --rounds 3
+        run: cd scripts/compare-perf && npm ci && npm run compare -- perf $GITHUB_SHA 6917e54eea4bb37d8beaca746cd5861a38331264 --tests-branch $GITHUB_SHA --rounds 3
```

I think we need to set this hash back to `trunk`:

```suggestion
        run: cd scripts/compare-perf && npm ci && npm run compare -- perf $GITHUB_SHA trunk --tests-branch $GITHUB_SHA --rounds 3
```

---

Sure
I did it intentionally, as I menthioned in the PR description - I replaced `trunk` temporary just for testing purposes, to make sure that CI is green and was going to provide the hash from this commit https://github.com/Automattic/studio/pull/2225 (Already pushed this change).